examples/oidc-iam-sa/index.ts

```diff
@@ -21,7 +20,7 @@ if (!cluster?.core?.oidcProvider) {
     throw new Error("Invalid cluster OIDC provider URL");
 }
 const clusterOidcProvider = cluster.core.oidcProvider;
-export const clusterOidcProviderUrl = clusterOidcProvider.url;
+export const clusterOidcProviderUrl = clusterOidcProvider.apply(u => u!.url);
```

Comment:
I think some users will stumble over this, we should probably include an example of this IRSA setup in the migration docs.